test/server/services/UrlManagementService.test.ts

```diff
@@ -0,0 +1,268 @@
+import { UrlManagementService } from '../../../src/server/services/UrlManagementService'
+import {
+  AlreadyExistsError,
+  AlreadyOwnLinkError,
+  NotFoundError,
+} from '../../../src/server/util/error'
+
+describe('UrlManagementService', () => {
+  const userRepository = {
+    findById: jest.fn(),
+    findByEmail: jest.fn(),
+    findOneUrlForUser: jest.fn(),
+    findUrlsForUser: jest.fn(),
+    findOrCreateWithEmail: jest.fn(),
+  }
+
+  const urlRepository = {
+    update: jest.fn(),
+    create: jest.fn(),
+    findByShortUrl: jest.fn(),
+    getLongUrl: jest.fn(),
+    plainTextSearch: jest.fn(),
+    // TODO: delete this, not needed
+    incrementClick: async () => {},
```

Comment:
Just checking if this line of code is meant to be merged

Reply:
It is - a later PR will surgically remove `UrlManagementServiceInterface.incrementClick()`